projects/zabbix/host_vars/dev.vagrant.localdomain/main.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Does a `dev` box get provisioned? If not, we should probably delete the vars folder from host_vars.